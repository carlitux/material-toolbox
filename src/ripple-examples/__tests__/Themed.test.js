// @flow
import React from 'react';
import { shallow } from 'enzyme';

import ThemedRipple from '../Themed';

describe('example::ThemedRipple', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<ThemedRipple />);
    expect(component).toMatchSnapshot();
  });
});
