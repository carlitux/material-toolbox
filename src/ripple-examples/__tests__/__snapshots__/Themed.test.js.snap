// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example::ThemedRipple Should match snapshoot 1`] = `
<Inner>
  <Cell
    desktop={6}
  >
    <Example
      source="import Ripple from 'material-toolbox/ripple';
import Elevation from 'material-toolbox/elevation';
import Theme from 'material-toolbox/theme';

const component = (
  <div>
    <Theme textStyle=\\"primary\\">
      <Elevation depth={2}>
        <Ripple primary>
          <div
            tabIndex=\\"0\\"
            className=\\"ripple-demo-surface\\">
            Primary
          </div>
        </Ripple>
      </Elevation>
    </Theme>

    <Theme textStyle=\\"secondary\\">
      <Elevation depth={2}>
        <Ripple accent>
          <div
            tabIndex=\\"0\\"
            className=\\"ripple-demo-surface\\">
            Secondary
          </div>
        </Ripple>
      </Elevation>
    </Theme>
  </div>
)"
      title="Theme Styles"
    >
      <Theme
        textStyle="primary"
      >
        <Elevation
          depth={2}
          transition={false}
        >
          <Ripple
            accent={false}
            cssOnly={false}
            disabled={false}
            primary={true}
            unbounded={false}
          >
            <div
              className="ripple-demo-surface"
              tabIndex="0"
            >
              Primary
            </div>
          </Ripple>
        </Elevation>
      </Theme>
      <Theme
        textStyle="secondary"
      >
        <Elevation
          depth={2}
          transition={false}
        >
          <Ripple
            accent={true}
            cssOnly={false}
            disabled={false}
            primary={false}
            unbounded={false}
          >
            <div
              className="ripple-demo-surface"
              tabIndex="0"
            >
              Secondary
            </div>
          </Ripple>
        </Elevation>
      </Theme>
    </Example>
  </Cell>
  <Cell
    desktop={6}
  >
    <Example
      source="import Ripple from 'material-toolbox/ripple';
import Elevation from 'material-toolbox/elevation';
import Theme from 'material-toolbox/theme';

const component = (
  <div>
    <Theme textStyle=\\"primary\\">
      <Elevation depth={2}>
        <Ripple primary cssOnly>
          <div
            tabIndex=\\"0\\"
            className=\\"ripple-demo-surface\\">
            Primary
          </div>
        </Ripple>
      </Elevation>
    </Theme>

    <Theme textStyle=\\"secondary\\">
      <Elevation depth={2}>
        <Ripple accent cssOnly>
          <div
            tabIndex=\\"0\\"
            className=\\"ripple-demo-surface\\">
            Secondary
          </div>
        </Ripple>
      </Elevation>
    </Theme>
  </div>
)"
      title="Theme Styles - CSS Only"
    >
      <Theme
        textStyle="primary"
      >
        <Elevation
          depth={2}
          transition={false}
        >
          <Ripple
            accent={false}
            cssOnly={true}
            disabled={false}
            primary={true}
            unbounded={false}
          >
            <div
              className="ripple-demo-surface"
              tabIndex="0"
            >
              Primary
            </div>
          </Ripple>
        </Elevation>
      </Theme>
      <Theme
        textStyle="secondary"
      >
        <Elevation
          depth={2}
          transition={false}
        >
          <Ripple
            accent={true}
            cssOnly={true}
            disabled={false}
            primary={false}
            unbounded={false}
          >
            <div
              className="ripple-demo-surface"
              tabIndex="0"
            >
              Secondary
            </div>
          </Ripple>
        </Elevation>
      </Theme>
    </Example>
  </Cell>
</Inner>
`;
